src/components/VisualizationPlugin/__tests__/VisualizationPlugin.spec.js

```diff
@@ -8,6 +8,19 @@ import * as options from '../../../modules/options.js'
 import ChartPlugin from '../ChartPlugin.js'
 import { VisualizationPlugin } from '../VisualizationPlugin.js'
 
+const consoleErrorSuppressor = (() => {
+    const originalConsoleError = console.error
+    const noop = () => {}
+    return {
+        suppress() {
+            console.error = noop
+        },
+        restore() {
+            console.error = originalConsoleError
+        },
+    }
+})()
```

Comment:
This helper allows us to temporarily suppress console errors. It works, but whether or not we should use it is another matter....